_posts/en/newsletters/2018-10-16-newsletter.md

```diff
@@ -0,0 +1,151 @@
+  transactions.  It was suggested that the simpler but more expensive
+  solution could be version 1, and the more complex but less expensive
+  solution could be version 2.
+
+- **Edge Dev++ talks published:** a two-day series of presentations from
+  leading Bitcoin contributors aimed at developers has been published
+  as [videos][dev vids] and [transcripts][dev transcripts].  The talks
+  cover the full range of topics, from introductory to advanced.  Three
+  talks may be especially interesting to Optech members:
+
+    1. *Exchange security* by Warren Togami.  Describes the causes
+       behind several notable major thefts from Bitcoin and altcoin
+       exchanges and lists a number of techniques businesses can use to
+       reduce their risk of loss.  ([Video][warren vid],
+       [transcript][warren transcript])
+
+    2. *Wallet Security, Key Management & Hardware Security Modules
+       (HSMs)]* by Bryan Bishop.  Suggests methods for decreasing the
+       risk that private keys will be stolen or misused.
+       ([Video][kanzure wallet vid], [transcript][kanzure wallet
+       transcript])
+
+    3. *Handling reorgs and forks* by Bryan Bishop.  Describes how to
+       secure your transactions against changes in the Bitcoin block
+       chain or consensus rules, including suggestions for how to test
+       your systems.  ([Video][kanzure reorg vid], no transcript)
+
+## CoreDev.tech
+
+CoreDev.tech is an invite-only event for well-known contributors to
+Bitcoin infrastructure projects such as Bitcoin Core and Lightning
+Network.  Discussions are not recorded, but Bryan Bishop helpfully
+writes rough and non-authoritative transcripts of some of the
+discussions during the events.  The following short summaries are based
+on some of the transcripts for the event in Tokyo last week:
+
+- **[Bitcoin Optech][optech transcript]:** Bitcoin Optech is introduced
+  and briefly discussed, followed by discussion of common problems
+  Bitcoin-using businesses encounter when using Bitcoin Core and other
+  open source infrastructure projects.
+
+- **[Using UTXO accumulators to reduce data storage
+  requirements][utreexo]:** Tadge Dryja describes work he's been doing
+  on UTXO accumulators that are similar in function to those described
+  in last week's newsletter but which have a different construction
+  based on hashes.  He further describes how they could be combined with
+  something like Cory Field's [UTXO Hash Set (UHO)][UHO] idea for full
+  nodes to store hashes of UTXOs instead of full UTXOs in order to
+  significantly reduce the amount of storage pruned full nodes would use
+  without necessarily requiring any changes to the consensus rules.
+
+- **[Script descriptors and DESCRIPT][]:** the backwards-compatible way
+  wallets such as Bitcoin Core default to watching for transaction
+  outputs paying them "is ambiguous, inflexible, and scales badly".
+  [Output script descriptors][] are a simple language for describing
```

Comment:
For script descriptors what about this link? https://github.com/bitcoin/bitcoin/blob/master/doc/descriptors.md

Reply:
This currently links to [sipa's gist](https://gist.github.com/sipa/e3d23d498c430bb601c5bca83523fa82) which I currently prefer because it describes the motivation for the language as well as providing a high-level overview.  As the language is refined beyond its initial design and becomes better implemented, I expect the document you linked in the Bitcoin Core repo to become superior and so we may switch links over to that then (to facilitate this, the link for the text "output script descriptors" is defined globally for the site in the `_include/references.md` file rather than in individual newsletters).